test/gtest/common/test_obj_size.cc

```diff
@@ -44,15 +44,20 @@ UCS_TEST_F(test_obj_size, size) {
    UCS_TEST_SKIP_R("Statistic enabled");
 #else
     EXPECTED_SIZE(ucp_ep_t, 104);
-    EXPECTED_SIZE(ucp_request_t, 200);
+    EXPECTED_SIZE(ucp_request_t, 224);
     EXPECTED_SIZE(uct_ep_t, 8);
     EXPECTED_SIZE(uct_base_ep_t, 8);
     EXPECTED_SIZE(uct_rkey_bundle_t, 24);
     EXPECTED_SIZE(uct_self_ep_t, 8);
     EXPECTED_SIZE(uct_tcp_ep_t, 72);
 #  if HAVE_TL_RC
+#    if IBV_EXP_HW_TM
```

Comment:
let's also add one for rc_mlx5_ep_t

Reply:
@yosefe to add this structure we have to include header which can't be compiled in C++ (there is void* arithmetics whica failed to compile by C++ compiler). to add it we have to refactor headers including. I will make it later

Reply:
@hoopoepg ok